chains/evm/calls/transactor/transact.go

```diff
@@ -0,0 +1,96 @@
+package transactor
+
+import (
+	"context"
+	"fmt"
+	"github.com/ChainSafe/chainbridge-core/chains/evm/calls/client"
+	"github.com/imdario/mergo"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/rs/zerolog/log"
+)
+
+type TransactOptions struct {
+	GasLimit uint64
+	GasPrice *big.Int
+	Value    *big.Int
```

Comment:
Would probably be nice to have nonce (*big.Int) and priority (string) for future use.

Reply:
I think this is something that we can add in the future, as this PR already has become too big to manage.